public/app/features/alerting/unified/components/rule-editor/QueryStep.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+import { Field, InputControl } from '@grafana/ui';
+import { RuleEditorSection } from './RuleEditorSection';
+import { useFormContext } from 'react-hook-form';
+import { RuleFormType, RuleFormValues } from '../../types/rule-form';
+import { ExpressionEditor } from './ExpressionEditor';
+import { GrafanaQueryEditor } from './GrafanaQueryEditor';
+import { isArray } from 'lodash';
+
+// @TODO get proper query editors in
+export const QueryStep: FC = () => {
+  const { control, watch, errors } = useFormContext<RuleFormValues>();
+  const type = watch('type');
+  const dataSourceName = watch('dataSourceName');
+  return (
+    <RuleEditorSection stepNo={2} title="Create a query to be alerted on">
+      {type === RuleFormType.system && dataSourceName && (
```

Comment:
what's the difference between these two editors?

Reply:
`GrafanaQueryEditor` is supposed to be the multi-query editor for grafana managed alerts,
`ExpressionEditor` is supposed to be loki or prometheus query editor for cloud alerts

If this doesn't make a lot of sense, let's change it later